src/Leaflet.Editable.js

```diff
@@ -1882,6 +1882,7 @@
         this.on('add', this._onEditableAdd);
     };
 
+    var flat = L.LineUtil._flat ? L.LineUtil._flat : L.Polyline._flat;  // 1.0/1.1 compat.
```

Comment:
It's a bit more tricky, because we want to support Leaflet < 1.1 (L.Polyline._flat), Leaflet == 1.1 (L.LinueUtil._flat) and Leaflet > 1.1 (L.LineUtil.isFlat) ;)

Fixed, anyway :)

I'll release as soon as we have a new Leaflet release.